src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
I like this more, but I have another suggestion: Could we try positioning the version string directly beneath the words Privacy Badger? Mockup attached below:

![screenshot from 2017-05-01 11 39 49](https://cloud.githubusercontent.com/assets/794578/25584499/3c6ccfd4-2e64-11e7-80c6-68d09b1d38a7.png)

Here are the suggested changes:
- Reduce size of "Privacy Badger"
- Move "Privacy Badger" up a little bit to make room for the version string below
- Align "Privacy Badger" and version string along left edge
- Create white space below the title to give some breathing room to the title and the summary text below the title